src/index.html

```diff
@@ -1,13 +1,61 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <section class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="card-image"
+        class="card__image"
+      >
+
+      <h1 class="card__product-name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h1>
+
+      <p class="card__product-code">
+        Product code: 195434
+      </p>
+
+      <div class="star-container">
+        <ul class="star-container__stars star-container__stars--4">
+          <li class="star-container__stars-star"></li>
+          <li class="star-container__stars-star"></li>
+          <li class="star-container__stars-star"></li>
+          <li class="star-container__stars-star"></li>
+          <li class="star-container__stars-star"></li>
+        </ul>
```

Comment:
use other tag instead of `<ul>`. `<div>`, for example. Because `<ul>` is semantically list, but it's not a list of stars, it's block of stars.